Connectors/Navisworks/Speckle.Connectors.NavisworksShared/Operations/Send/NavisworksRootObjectBuilder.cs

```diff
@@ -35,6 +37,10 @@ public RootObjectBuilderResult Build(
     CancellationToken cancellationToken
   )
   {
+#if DEBUG
+    // This is a temporary workaround to disable node merging for debugging purposes - false is default, true is for debugging
+    SkipNodeMerging = false;
```

Comment:
seems marked as `false` on debugging 🤔 but it has no effect for sure on production

Reply:
This is intentional. It happens to be the last time I ran it I set it as false because I didn't want to test for that. I know it's something that I want to test in the future again this Pragma is literally so I don't accidentally leave it set to true for production. 

Long term I'll remove it.